added tooltips, and removed unused files and some log messages.

lutzmor opened the pull request:
Added tooltips to all possible spells. Open are the combined things like talents etc, will update the issue maybe we could discuss what to do with them :).

lutzmor committed:
added tooltips, and removed unused files and some log messages.

seanpeters86 commented on beta/js/internal/Chart/ChartHelper.js:
```diff
@@ -5,15 +5,34 @@ function buildWowheadTooltips(data, breakConidition, simsBtn) {
+    
     if (id == null) {
-      id = "";
+      /*
+       * Current fix ( the sims don't return a id here at the moment)
+       */
+      if(dpsName.includes("Shadowflame_Prism")) {
+        id = data[jsonIds]["Shadowflame_Prism"];
+      } else if (dpsName.includes("Twins_of_the_Sun_Priestess")) {
+        id = "336897";
+      } else {
+        id = "";
+      }
```
ive found the problem causing this and will fix on the next run, but we should handle this generically instead of hard coding these options. If I don't have it the value will always be `null` on my side (could change this if we want).

lutzmor replied:
Yeah, was "just" a temporary thing until we get the data from sl-shadow-priest right. I reverted the change.

lutzmor committed:
update for bugs

lutzmor committed:
delete files

lutzmor committed:
reverted legendarie tooltip fix

lutzmor commented:
I added another "bigger" commit to this, i resolved some problems with the trinket charts and charts in general. What i included fixing is the following:

- Issue 138 ( Chart barr colors are having a party )
- Issue 67 ( Racials Chart Links to Item URLs on Wowhead for tooltips )
- Issue 64 ( Partial i added icons in the first step, need to fix that with data from wowhead )
- Issue 62 ( Update chart to be able to share links to specific charts )
- Issue 61 ( Fix wowhead links on the beta side )

And i added two empty files in prepare of Issue 60 and Issue 139.

lutzmor committed:
update parameter, to work for current page layout.

lutzmor committed:
added aggregate view for covenant choices
